scripts/badger/process_bribes_graviaura.py

```diff
@@ -47,6 +47,9 @@ def claim_and_sell_for_weth(claim_only=False):
     # NOTE: badger is directly emitted by the strat to tree
     # NOTE: aura is sent to processor, but should not be sold for weth
     for addr, mantissa in claimed.items():
+        if addr == "0x0":
```

Comment:
Aren't ETH bribes shown as the `hhBribeVault`: https://etherscan.io/address/0x7c2a951d062cc7b7c6f9c7aa7e80a6f20eaa8cab#code#L2069?

According to the [rewards_distributor](https://etherscan.io/address/0x0b139682D5C9Df3e735063f46Fb98c689540Cf3A#readContract), it is: [0x9DDb2da7Dd76612e0df237B89AF2CF4413733212](https://etherscan.io/address/0x9DDb2da7Dd76612e0df237B89AF2CF4413733212)

Reply:
i believe the endpoint which is the one we rely on for the claims here uses different convention for eth

Reply:
yea the hhand.xyz api endpoint uses `"0x0"` for naked eth

Reply:
Got it, I figured it could be the case but couldn't confirm myself. LGTM then.